src/image-tags.js

```diff
@@ -54,6 +54,34 @@ function pickImage(pageType, data, url) {
   }
 }
 
+function pickAttributionFromCard (story, cardId) {
+  const { metadata = {} } = story.cards.find(card => card.id === cardId) || {};
+  if(metadata && !isEmpty(metadata) && get(metadata, ['social-share', 'image', 'attribution'], false)) {
+    return metadata['social-share'].image.attribution || undefined;
+  }
+}
+
+function pickAttributionFromStory(story){
+  return story["hero-image-attribution"] || story.summary || get(story, ["alternative", "home", "default", "headline"]) || story.headline;
+}
+
```

Comment:
try to get it from `pickImage`

Reply:
do not want to touch existing other feature, let me try that.

Reply:
👍